### PR TITLE
Docs 1981 SCALE Root SSH Test

### PR DESCRIPTION
**Pull request by q5sys**

### CODE REVIEW

File: tests/bdd/conftest.py [hunk 1]
```diff
@@ -76,8 +76,8 @@ def driver():
 
 
 # Close firefox after all tests are completed
-def pytest_sessionfinish(session, exitstatus):
-    web_driver.quit()
+# def pytest_sessionfinish(session, exitstatus):
```
Comment (ericbsd) [inline tests/bdd/conftest.py:79]: Please uncomment lines 79 and 80 to ensure that firefox gets closed after tests are done on bob.

**Review by ericbsd — CHANGES_REQUESTED** [review]

File: tests/bdd/conftest.py (hunk 1 above)
Comment (q5sys) [reply tests/bdd/conftest.py:79]: good catch, I forgot to do that.  Just pushed a new commit to uncomment those.

Comment (q5sys) [reply tests/bdd/conftest.py:79]: wait, sorry this is a different file. I'll fix that in a second

**Review by ericbsd — APPROVED** [review]
LGTM

**Pull request merged**